nvidia_triton/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "72d17043-fa30-4f5c-95cb-939906d86fb7",
+  "app_id": "nvidia-triton",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "NVIDIA Triton Inference Server is open source inference-serving software",
```

Comment:
nit 

```suggestion
    "description": "NVIDIA Triton Inference Server is an open source inference-serving software",
```

Reply:
I just modified it because of a previous comment : https://github.com/DataDog/integrations-core/pull/15991#discussion_r1378206175

Reply:
🤔 That's what they have on their github repo, but ok then!